packages/blockchain-ui/src/theme/theme.ts

```diff
@@ -19,8 +21,9 @@ export const createTheme = ({
   media = {},
   shadows = {},
   typography = {},
+  colorScheme = "light",
```

Comment:
no need for colorScheme . if you want to create a dark theme . just use createTheme function . by passing overrides to it example createTheme({palette : {common: {main: "newcolor"}};

Reply:
I have made it override ..